src/utils/Config.tsx

```diff
@@ -13,17 +14,18 @@ export const baseStrokeThickness = 1;
 export const baseTranslateOffset = 12;
 export const baseZoomThreshold = 16;
 export const defaultMarkerColor = "#3786C2";
-export const evenRowColor = "#ededed";
 export const highlightedMarkerColor = "#8bc5f1";
-export const highlightedRowColor = "#c3e3fb";
 export const maxZoom = 128;
 export const minZoom = 1;
-export const oddRowColor = "#f6f6f6";
-export const tableHeaderColor = "#dcdcdc";
 export const zoomMultiplier = 2;
+export const zoomForFullName = 64;
```

Comment:
Related to "Display "X installations" as marker text for markers with 2+ installations"